examples/beam/without_job_server/beam_wordcount_java.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: beam-wordcount-java
+spec:
+  template:
+    metadata:
+      labels:
+        app: beam-wordcount-java
+    spec:
+      containers:
+        - name: beam-wordcount-java
+          image: apache/beam_java_sdk
```

Comment:
Can we pin to a specific version to prevent it from breaking in the future?

Reply:
Solved in current commit